example/test_problem/ELBDM/PlaneWave/README

```diff
@@ -0,0 +1,16 @@
+Compilation flags:
+========================================
+Enable : MODEL=ELBDM, FLOAT8
+Disable: GRAVITY, PARTICLE
+
+
+Default setup:
+========================================
+1. Evolve the plane wave for six periods
+2. Apply the analytical solution as user-defined BC
+   --> Set OPT__BC_FLU_* = 4
```

Comment:
Revise it if you decide to use periodic BC.

Reply:
Done.